tree/20_cfe_basics/cfengine/packages.cf

```diff
@@ -1163,7 +1163,7 @@ body package_method yum_rpm_enable_repo(repoid)
 # Based on `yum_rpm()` with addition to enable a repository for the install.
```

Comment:
I think we should exclude the ` tree/20_cfe_basics/cfengine/` folder as it is synchronized from cfengine